script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -549,7 +549,7 @@ I strike their weapon, sending it flying.
 -- 一歩踏み込む。無防備な胸の“線”を断つべく、牙のように短刀を突き立てる。
 // 牙 modulated as a verb, since 'thrusting like a fang' doesn't really parse
 I take a step forward.
- The 'line' I must cut is right there on his chest, defenseless.
+ The 'line' I must cut is right there on their chest, defenseless.
```

Comment:
I'd revert this one as well. The subject isn't even specified in the JP, we have to spirit it and may as well use the more scoped term.